base_comment_template/tests/test_base_comment_template.py

```diff
@@ -15,19 +16,20 @@ class TestCommentTemplate(common.SavepointCase):
     def setUpClass(cls):
         super().setUpClass()
         setup_test_model(cls.env, ResUsers)
-        cls.user_obj = cls.env["ir.model"].search([("model", "=", "res.users")])
-
+        cls.user_obj = cls.env.ref("base.model_res_users")
         cls.user = cls.env.ref("base.user_demo")
         cls.user2 = cls.env.ref("base.demo_user0")
         cls.partner_id = cls.env.ref("base.res_partner_12")
         cls.partner2_id = cls.env.ref("base.res_partner_10")
-        cls.company = cls.env["res.company"].search([], limit=1)
+        cls.main_company = cls.env.ref("base.main_company")
+        cls.company = cls.env["res.company"].create({"name": "Test company"})
```

Comment:
Please do no create new company, as other problem may arise (required fields not filled, etc).

Reply:
In these use case i need to have created 2 companies, maybe `.copy()` it's a good idea?

Reply:
Why 2 companies are needed? Maybe regression tests are going too deep... Anyway, `copy` can prevent such problem, but I prefer to not require 2 companies if possible.

Reply:
Ok, i have removed everything about it.